lendingbot.py

```diff
@@ -26,562 +32,616 @@
+        print '\ndefault.cfg.example has been copied to default.cfg\nEdit it with your API key and custom settings.\n'
+        raw_input("Press Enter to acknowledge and exit...")
+        exit(0)
+
+    if config.has_option('BOT', 'sleeptimeactive') and config.has_option('BOT', 'sleeptimeinactive'):
+        sleepTimeActive = float(config.get("BOT", "sleeptimeactive"))
+        sleepTimeInactive = float(config.get("BOT", "sleeptimeinactive"))
+    else:
+        sleepTime = float(config.get("BOT", "sleeptime"))  # If it can't find a setting, run with the old cfg.
+        sleepTimeActive = sleepTime
+        sleepTimeInactive = sleepTime
+        print "!!! Please update to new config that includes Inactive Mode. !!!"  # Update alert.
+    apiKey = config.get("API", "apikey")
+    apiSecret = config.get("API", "secret")
+    minDailyRate = Decimal(config.get("BOT", "mindailyrate")) / 100
+    maxDailyRate = Decimal(config.get("BOT", "maxdailyrate")) / 100
+    spreadLend = int(config.get("BOT", "spreadlend"))
+    gapBottom = Decimal(config.get("BOT", "gapbottom"))
+    gapTop = Decimal(config.get("BOT", "gaptop"))
+    if config.has_option("BOT", "sixtyDayThreshold"):
+        sixtyDayThreshold = float(config.get("BOT", "sixtydaythreshold")) / 100
+        xDayThreshold = sixtyDayThreshold
+        xDays = "60"
+    else:
+        xDayThreshold = Decimal(config.get("BOT", "xdaythreshold")) / 100
+        xDays = str(config.get("BOT", "xdays"))
+    auto_renew = int(config.get("BOT", "autorenew"))
+    if config.has_option('BOT', 'outputCurrency'):
+        outputCurrency = config.get('BOT', 'outputCurrency')
+    if config.has_option('BOT', 'maxtolent'):
+        maxtolent = Decimal(config.get('BOT', 'maxtolent'))
+    if config.has_option('BOT', 'maxpercenttolent'):
+        maxpercenttolent = Decimal(config.get('BOT', 'maxpercenttolent')) / 100
+    if config.has_option('BOT', 'maxtolentrate'):
+        maxtolentrate = Decimal(config.get('BOT', 'maxtolentrate')) / 100
+    if config.has_option('BOT', 'minloansize'):
+        minLoanSize = Decimal(config.get("BOT", 'minloansize'))
+    if config.has_option('BOT', "transferableCurrencies"):
+        transferableCurrencies = (config.get("BOT", "transferableCurrencies")).split(",")
+    else:
+        transferableCurrencies = []
+
+    if config.has_option("BOT", "coinconfig"):
+        try:
+            # parsed
+            coinconfig = (json.loads(config.get("BOT", "coinconfig")))
+            for cur in coinconfig:
+                cur = cur.split(':')
+                coincfg[cur[0]] = dict(minrate=(Decimal(cur[1])) / 100, maxactive=Decimal(cur[2]),
+                                       maxtolent=Decimal(cur[3]), maxpercenttolent=(Decimal(cur[4])) / 100,
+                                       maxtolentrate=(Decimal(cur[5])) / 100)
+        except Exception as e:
+            print "Coinconfig parsed incorrectly, please refer to the documentation. Error: " + e
```

Comment:
I did not change logic in this PR, it was just marked a violation that we had an exception handler that did nothing so I added a print to notify the user.